files/scripts/healdata/heal-cedar-data-ingest.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import json
+import requests
+import os
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--directory", "-d", help="Cedar Directory ID for registering ")
+parser.add_argument("--access_token", "-a", help="User access token")
+
+args = parser.parse_args()
+
+if not args.directory:
+    print("Directory ID is required!")
+	exit(1)
+if not args.access_token:
+	print("User access token is required!")
+	exit(1)
+
+dir_id = args.directory
+access_token = args.access_token
+
+token_header = {"Authorization": 'access_token ' + access_token}
+
+# Get the metadata from cedar to register
+cedar = requests.get("http://revproxy-service/cedar/get_directory/" + dir_id, headers=token_header)
+
+# If we get metadata back now register with MDS
+if cedar.status_code == 200:
+	print("successfully got records from cedar directory")
+
+	metadata_return = cedar.json()
+
+	for i in metadata_return:
+		print(i["Metadata Location"]["Metadata Location - Details"]["nih_application_id"])
+
+		# Get the metadata record for the nih_application_id
+		mds = requests.get("http://revproxy-service/mds/metadata/" + i["Metadata Location"]["Metadata Location - Details"]["nih_application_id"],
+			headers=token_header
+		)
+		if mds.status_code == 200:
+			mds_res = mds.json()
+			if mds_res["_guid_type"] == "discovery_metadata":
+				print("Metadata is already registered. Skipping this instance")
```

Comment:
if id exists in MDS, the existing record should get updated with field values being pulled in from CEDAR

Reply:
add lines 47-49 to add data coming from cedar

Reply:
I think L47-49 change are good, but for this particular case, if `mds_res["_guid_type"] == "discovery_metadata"`, then this script should update the registered study metadata with new information pulled from CEDAR instances.

This script is not only for unregistered metadata, it should also update   registed metadata